minecraft-server/start-minecraft.sh

```diff
@@ -33,19 +33,35 @@ esac
 
 cd /data
 
-echo "Checking minecraft / forge type information."
+echo "Checking type information."
 case "$TYPE" in
-  VANILLA)
-    SERVER="minecraft_server.$VANILLA_VERSION.jar"
-
-    if [ ! -e $SERVER ]; then
-      echo "Downloading $SERVER ..."
-      wget -q https://s3.amazonaws.com/Minecraft.Download/versions/$VANILLA_VERSION/$SERVER
-    fi
-  ;;
+  *BUKKIT|*bukkit|SPIGOT|spigot)
+    TYPE=SPIGOT
```

Comment:
Got it. I remember there was controversy with Bukkit...and then I hadn't researched much of Spigot. Thanks.